content/webhooks/index.md

```diff
@@ -84,9 +84,7 @@ organization (`organization`) and/or repository (`repository`) which the event
 occurred on.
 
 {{#tip}}
-
 **Note:** Payloads are capped at 5 MB. If your event generates a larger payload, a webhook will not be fired. This may happen, for example, on a `create` event if many branches or tags are pushed at once. We suggest monitoring your payload size to ensure delivery.
-
```

Comment:
I think you should update Hubot to not apply the absolute difference, but just the diff of the commits that were upstream.
(Like cherry-pick 'n squash with the `sync changes from upstream` message)